extensions/amp-analytics/0.1/test/test-vendors.js

```diff
@@ -155,10 +155,20 @@ describes.realWin(
                   .stub(ExpansionOptions.prototype, 'getVar')
                   .callsFake(function(name) {
                     let val = this.vars[name];
+                    // Vendor defined variable
                     if (val == null || val == '') {
                       val = '!' + name;
                     }
-                    return val;
+                    // amp-analytics variable
+                    switch (name) {
```

Comment:
Why we need to handle the variable specially?

Reply:
The test expands the variables out so `TIMEZONE_CODE` -> `America%2FLos_Angeles` instead of `_timezone_code_`.

So need to whitelist them here.

Other option would be to change everyone's vendor configs, but that wouldn't work, unless we faked all of these variables' expansions (i.e. faking TIMEZONE_CODE ->  America%2FLos_Angeles, no matter where you're testing from).

Reply:
Feels like we should start to mock those variables from `getMacros()` just like variables from the global list now.

Reply:
Sounds like the right approach. Will do that first.